lib/pbench/agent/tool_meister.py

```diff
@@ -177,6 +179,42 @@ def start(self):
 
             args = [f"python2 {script_path}"]
             self.process = subprocess.Popen(args, shell=True)
+        elif self.name == "pcp":
+            self.logger.debug("PMCD STARTUP")
+            if not find_executable("podman"):
+                self.logger.error(
+                    "Podman is not installed on this system (required by some registered tools, aborting launch)"
+                )
+                self.failure = True
+                return 0
+
+            with open("pcp-meister.log", "w") as pcp_logs:
+                args = ["podman", "pull", "quay.io/meyceoz/pcp-exposer"]
+                try:
+                    pcp_pull = subprocess.Popen(args, stdout=pcp_logs, stderr=pcp_logs)
+                    pcp_pull.wait()
+                except Exception as exc:
+                    self.logger.error("Podman pull process failed: '%s'", exc)
+                    self.failure = True
+                    return 0
+                self.podname = "exposer" + datetime.now().strftime("%m-%d-%y-%H-%M-%S")
+                args = [
+                    "podman",
+                    "run",
+                    "--network",
+                    "host",
+                    "--name",
+                    self.podname,
+                    "quay.io/meyceoz/pcp-exposer",
```

Comment:
Not sure about the hard-coded path here.  Can we make that configurable?

And consider a name change?  It seems like we would want to use a name closer the function of what the containers are doing.  Perhaps `pbench-pmcd` and `pbench-pmlogger`.